.github/workflows/prod_release.yml

```diff
@@ -54,5 +54,10 @@ jobs:
         env:
           NODE_AUTH_TOKEN: ${{ secrets.NPM_TOKEN }}
 
+      - name: Deprecate Solar Theme on npm
+        run: npm deprecate @refinitiv-ui/solar-theme "Solar theme no longer supported. Use @refinitiv-ui/halo-theme."
```

Comment:
The message should be "Solar theme is deprecated. Consider migrating to @refinitiv-ui/halo-theme instead."